packages/uploadthing/src/internal/upload-builder.ts

```diff
@@ -35,7 +35,11 @@ function internalCreateBuilder<
       },
     },
 
-    inputParser: { parse: () => ({}), _input: {}, _output: {} },
+    inputParser: {
+      parse: () => undefined,
+      _input: undefined,
+      _output: undefined,
+    },
```

Comment:
we previously had a type-mismatch when no input parser was set. No big deal but this makes sure the type and value match